main.py

```diff
@@ -2,6 +2,35 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.all().delete()
+    Actor.objects.all().delete()
+
+    Genre.objects.create(name = "Western",)
+    Genre.objects.create(name ="Action", )
+    Genre.objects.create(name = "Dramma", )
+    Actor.objects.create(first_name = "George", last_name = "Klooney")
+    Actor.objects.create(first_name = "Kianu", last_name = "Reaves")
+    Actor.objects.create(first_name = "Scarlett", last_name = "Keegan")
+    Actor.objects.create(first_name = "Will", last_name = "Smith")
+    Actor.objects.create(first_name = "Jaden", last_name ="Smith")
+    Actor.objects.create(first_name ="Scarlett", last_name = "Johansson")
+
+    Genre.objects.filter(name = "Dramma").update(name = "Drama")
+    Actor.objects.filter(first_name = "George", last_name = "Klooney").update(
+        last_name = "Clooney")
```

Comment:
There is a typo in the actor's last name 'Klooney'. It should be 'Clooney'. This is corrected later in the script, but it's better to avoid the typo initially.